tests/framework/clients/rest_client.go

```diff
@@ -75,7 +71,21 @@ func CreateRestAPIClient(platform enums.RookPlatformType, k8sHelper *utils.K8sHe
 	}
 	client := rclient.NewRookNetworkRestClient(endpoint, httpclient)
 
-	return &RestAPIClient{client}
+	//make sure rest client is up and available
+	inc := 0
+	for inc < 20 {
```

Comment:
There are a lot of these loops where we retry an operation. Consider:
- define consts for the retry count and sleep time
- write to the log so we know what is happening in the test. Otherwise it just looks like a hang when something goes wrong